tests/actions/test_core_actions.py

```diff
@@ -868,14 +868,29 @@ def test_APINotificationAction_run(self, checkpoint_result: CheckpointResult):
         )
 
     @pytest.mark.unit
-    @pytest.mark.xfail(
-        reason="Not yet implemented for this class", strict=True, raises=NotImplementedError
-    )
     def test_EmailAction_run(self, checkpoint_result: CheckpointResult):
         action = EmailAction(
-            smtp_address="test", smtp_port=587, receiver_emails="test1@gmail.com, test2@hotmail.com"
+            smtp_address="test",
+            smtp_port="587",
+            receiver_emails="test1@gmail.com, test2@hotmail.com",
         )
-        action.v1_run(checkpoint_result=checkpoint_result)
+
+        with mock.patch("great_expectations.checkpoint.actions.send_email") as mock_send_email:
+            out = action.v1_run(checkpoint_result=checkpoint_result)
+
+        mock_send_email.assert_called_once_with(
+            title=mock.ANY,
```

Comment:
I think ANY makes sense here, but I might add another assertion below to verify that the argument passed kind of looks like the right string. (e.g. `suite_name in call_args.kwargs["title"]` (or whatever the syntax is for that))